### PR TITLE
chore: Update to Angular7

### PR DESCRIPTION
**Pull request by alippai**

### CODE REVIEW

File: projects/ngx-translate/http-loader/package.json [hunk 1]
```diff
@@ -20,8 +20,8 @@
   },
   "peerDependencies": {
     "@ngx-translate/core": ">=10.0.0",
-    "@angular/common": ">=6.0.0",
-    "rxjs": ">=6.0.0"
+    "@angular/common": ">=7 .0.0",
```
Comment (ocombe) [inline projects/ngx-translate/http-loader/package.json]:
```suggestion
    "@angular/common": ">=7.0.0",
```

File: projects/ngx-translate/http-loader/package.json [hunk 2]
```diff
@@ -20,8 +20,8 @@
   },
   "peerDependencies": {
     "@ngx-translate/core": ">=10.0.0",
```
Comment (ocombe) [inline projects/ngx-translate/http-loader/package.json]:
```suggestion
     "@ngx-translate/core": ">=11.0.0",
```

**Review by ocombe — CHANGES_REQUESTED** [review]
I added some suggestions

**Comment by alippai** [comment]
Fixed & rebased

**Review by ocombe — DISMISSED** [review]
thanks a lot!

**Comment by alippai** [comment]
@ocombe cannot add new package-lock.json as there is no ngx-translate/core@11.0.0 released

**Comment by ocombe** [comment]
I just published it, can you retry please?

**Comment by alippai** [comment]
@ocombe done 🚀

**Review by ocombe — APPROVED** [review]

**Comment by ocombe** [comment]
Thanks a lot for your help!

**Pull request merged**